lib/handlers/get.js

```diff
@@ -176,7 +176,7 @@ function globHandler (req, res, next) {
       var data = $rdf.serialize(
         undefined,
         globGraph,
-        null,
+        filename,
         'text/turtle')
```

Comment:
From what I understand, the notion of "globbing all the graphs in an LDP container into a single graph" is agnostic to a serialization format.